app/org/sagebionetworks/bridge/models/studies/ConsentSignature.java

```diff
@@ -54,33 +68,51 @@ private ConsentSignature(@JsonProperty("name") String name, @JsonProperty("birth
      * @param imageData
      *         image data as a Base64 encoded string, optional
      * @param imageMimeType
-     *         image MIME type (ex: image/png), optioanl
+     *         image MIME type (ex: image/png), optional
      * @return validated ConsentSignature
      * @throws InvalidEntityException
      *         if called with invalid fields
      */
     public static ConsentSignature create(@Nonnull String name, @Nonnull String birthdate, @Nullable String imageData,
-            @Nullable String imageMimeType) throws InvalidEntityException {
-        ConsentSignature sig = new ConsentSignature(name, birthdate, imageData, imageMimeType);
+            @Nullable String imageMimeType, @Nonnull long signedOn) throws InvalidEntityException {
+        ConsentSignature sig = new ConsentSignature(name, birthdate, imageData, imageMimeType, signedOn);
         Validate.entityThrowingException(VALIDATOR, sig);
         return sig;
     }
```

Comment:
We should consider refactoring this class to use the Builder pattern:
1. We can remove the redundant create() method.
2. The builder could subsitute "now" for signedOn if it's not specified (which will always be the case for old clients). I feel like this is cleaner than passing in signedOn for every request.
3. Almost all the attributes are strings. Builder pattern makes it less likely that these get mixed up.